scripts/data-retention-remove-supplier-declarations.py

```diff
@@ -43,6 +44,7 @@
 
     dry_run = arguments['--dry-run']
     verbose = arguments['--verbose']
+    user = arguments['<user>']
```

Comment:
Let's set a default with `getpass` in case the arg is missing (see https://github.com/alphagov/digitalmarketplace-scripts/blob/master/scripts/insert-framework-results.py#L35 for an example).

Reply:
Ugh. And I did this on the other one as well. Good spot, thank you!